src/components/Home/Home.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
You haven't use `props` or `state` anywhere. Get rid of them and if possible make this component a functional component.